.circleci/src/jobs/check.yml

```diff
@@ -2,9 +2,9 @@ docker:
   - image: cimg/node:14.19.0
 steps:
   - checkout
-  - check_config
   - restore_yarn_cache
   - restore_yarn_tools_cache
+  - check_config
```

Comment:
Any specific reason why you changed the order here? If not, I'd prefer to keep the old order since builds will fail faster/use less credits if the config is wrong.

Reply:
not possible: Couldn't find the node_modules state file - running an install might help. I tried looking for a solution, but didn't really find sth but placing node <command> directly into the ci. Maybe you want to take a look?

Reply:
No, I think it is fine. Shouldn't happen often anyway :)